frontend/cypress/tests/chartView/chartView_filterBreakdown.js

```diff
@@ -0,0 +1,59 @@
+describe('filter breakdown', () => {
+  it('check breakdown by file type should show file types', () => {
+    Cypress.wait();
+
+    cy.get('#summary label.filter-breakdown input:visible')
+        .should('be.visible')
+        .check()
+        .should('be.checked');
+
+    // should show file type checkboxes
+    cy.get('#summary div.fileTypes')
+        .should('be.visible');
+
+    // should show file type legends
+    cy.get('#summary-charts div.summary-charts__fileType--breakdown')
+        .should('be.visible');
+  });
+
+  it('uncheck all file types should show no file types', () => {
+    Cypress.wait();
+
+    cy.get('#summary label.filter-breakdown input:visible')
+        .should('be.visible')
+        .check()
+        .should('be.checked');
+
+    // uncheck all file types
+    cy.get('#summary div.fileTypes input:visible')
+        .first()
+        .uncheck()
+        .should('not.be.checked');
+
+    // should not show file type legends
+    cy.get('#summary-charts div.summary-charts__fileType--breakdown')
+        .should('not.be.visible');
+  });
+
+  it('uncheck file type should uncheck all option and not show legend', () => {
+    Cypress.wait();
```

Comment:
Can you add a test for - if one of the checkboxes is not checked, the `All` checkboxes should also not be checked?

Reply:
done